### PR TITLE
add glove_embedding.py to construct the glove wordtovector

### PR DESCRIPTION
**Pull request by liusiye**
modified the lstm.py to read wordtovector

### CODE REVIEW

**Comment by dennybritz** [comment]
Thanks, I'll need to look at this in detail later. Have you tried running it and see if gives any accuracy improvements?

**Comment by liusiye** [comment]
hi,denny:
when I only use wordtovector, it only improved about 5% in 10@5.
but , now I'm trying to use RNN output as context with learning_rate=0.01(still didn't git push,I will push it later).
the result is much better ,10@1:40%, 10@2 57%, 10@5 83%
Now I'm checking the paper, try to figure out it.

2016-04-28 

18010076513 



发件人：Denny Britz <notifications@github.com>
发送时间：2016-04-28 22:44
主题：Re: [dennybritz/chatbot-retrieval] add glove_embedding.py to construct the glove wordtovector (#2)
收件人："dennybritz/chatbot-retrieval"<chatbot-retrieval@noreply.github.com>
抄送："liusiye"<18010076513@163.com>,"Author"<author@noreply.github.com>

Thanks, I'll need to look at this in detail later. Have you tried running it and see if gives any accuracy improvements?

**Comment by dennybritz** [comment]
Thanks for trying these things, very useful! I'm also running some experiments now. Let's see how that goes.
